site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -196,6 +196,8 @@ include::_tenant-generator-length-config-rules.adoc[]
 :generator_type!:
 :generator_display_name!:
 +
+[field]#tenant.externalIdentifierConfiguration.samlv2AuthNRequestIdTimeToLiveInSeconds# [type]#[Integer]# [optional]#Optional# [default]#300# [since]#Available since 1.20.0#::
```

Comment:
Good catch, looks like I missed this one from a previous release. This came out in `1.19.0` not `1.20.0`. 
https://fusionauth.io/docs/v1/tech/release-notes/#version-1-19-0